py-core/setup_workspace.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Pull Tezos Docker Image
+docker pull tezos/tezos:latest-release
+
+# Pull Redis Docker Container for Python Job Queue
+docker pull redis
+
+# Build Mainnet Arronax Image Locally
+git clone https://github.com/Cryptonomic/Arronax.git
+
+cd Arronax
+
+touch src/config.tsx
+
+echo "import { Config } from './types';
+
+const configs: Config[] = [
+  {
+    platform: 'tezos',
+    network: 'mainnet',
+    displayName: 'Tezos Mainnet',
+    url: 'http://conseil-api:80',
+    apiKey: 'conseil',
+    nodeUrl: 'http://tezos-node:8732',
+    entities: ['blocks', 'operations', 'accounts', 'bakers', 'governance'],
+    hiddenEntities: ['originated_account_maps', 'big_maps', 'big_map_contents']
+  }
+]
+
+export default configs;
+" >> src/config.tsx
+
+docker build -t arronax-mainnet .
+
+# shellcheck disable=SC2103
+cd ..
+
+rm -rf Arronax
+
+# Build Carthagenet Arronax Image Locally
+git clone https://github.com/Cryptonomic/Arronax.git
+
+cd Arronax
+
+touch src/config.tsx
+
+echo "import { Config } from './types';
+
+const configs: Config[] = [
+  {
+    platform: 'tezos',
+    network: 'carthagenet',
+    displayName: 'Tezos Carthagenet',
+    url: 'http://conseil-api:80',
+    apiKey: 'conseil',
+    nodeUrl: 'http://tezos-node:8732',
+    entities: ['blocks', 'operations', 'accounts', 'bakers', 'governance'],
+    hiddenEntities: ['originated_account_maps', 'big_maps', 'big_map_contents']
+  }
+]
+
+export default configs;
+" >> src/config.tsx
```

Comment:
Since you use it in multiple places, I'd put this snippets into a file and use the output from `cat`. You can deal with the differences in network using `grep`

Reply:
That's how I did it before, but we don't want to rely on bash scripts, and leave it to Python as much as possible 
(the only reason I'm doing it this way is cause docker-compose doesn't have an API)